ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlRestConfig.java

```diff
@@ -122,6 +152,42 @@
         5000L,
         Importance.LOW,
         KSQL_HEALTHCHECK_INTERVAL_MS_DOC
+    ).define(
+        KSQL_HEARTBEAT_SEND_INTERVAL_MS_CONFIG,
+        Type.LONG,
+        100L,
```

Comment:
Let's add a config that disable HeartbeatHandler completely and by default make it `false`? We still don't understand how pull queries and heartbeats over the same RPC layer will interplay.. This way, we can safely experiment, stabilize, tune these configs and then enable.